src/Controller/TideApiController.php

```diff
@@ -153,10 +153,16 @@ public static function create(ContainerInterface $container) {
    *   JSON response.
    */
   public function getRoute(Request $request) {
+    global $base_url;
     $code = Response::HTTP_NOT_FOUND;
     $entity = NULL;
 
     $path = $request->query->get('path');
+    $query = [];
```

Comment:
For consistency I think you can do `$query = $request->query->all();` https://stackoverflow.com/questions/11227285/getting-all-request-parameters-in-symfony-2 for reference